recipes/importmagic/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "importmagic" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "3f7757a5b74c9a291e20e12023bb3bf71bc2fa3adfb15a08570648ab83eaf8d8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - importmagic
+
+about:
+  home: https://github.com/alecthomas/importmagic
+  license: BSD2
```

Comment:
Please add a comment noting the aforementioned PR with `license_file: LICENSE` commented out below.